app/uk/gov/hmrc/apidocumentation/views/AuthorisationUserRestrictedEndpointsView.scala.html

```diff
@@ -482,6 +486,30 @@ <h5 class="govuk-heading-s faded-text">Error scenarios</h5>
                         <td class="govuk-table__cell"><code class="code--slim">invalid_request</code></td>
                         <td class="govuk-table__cell"><code class="code--slim">code is invalid</code></td>
                     </tr>
+                    <tr class="govuk-table__row">
+                        <td class="govuk-table__cell">PKCE code_verifier wasn't expected as no code_challenge was provided when requesting an authorization code</td>
+                        <td class="govuk-table__cell"><code class="code--slim">400</code> (Bad Request)</td>
+                        <td class="govuk-table__cell"><code class="code--slim">invalid_request</code></td>
+                        <td class="govuk-table__cell"><code class="code--slim">code_verifier is not expected</code></td>
+                    </tr>
+                    <tr class="govuk-table__row">
+                        <td class="govuk-table__cell">PKCE code_verifier was expected but none was supplied</td>
+                        <td class="govuk-table__cell"><code class="code--slim">400</code> (Bad Request)</td>
+                        <td class="govuk-table__cell"><code class="code--slim">invalid_request</code></td>
+                        <td class="govuk-table__cell"><code class="code--slim">code_verifier is expected when code_challenge was supplied</code></td>
+                    </tr>
+                     <tr class="govuk-table__row">
+                        <td class="govuk-table__cell">PKCE code_verifier is made up of fewer than 43, more than 128, or invalid characters</td>
+                        <td class="govuk-table__cell"><code class="code--slim">400</code> (Bad Request)</td>
+                        <td class="govuk-table__cell"><code class="code--slim">invalid_request</code></td>
+                        <td class="govuk-table__cell"><code class="code--slim">code_verifier must contain valid characters of length between 43 and 128</code></td>
```

Comment:
Which characters are valid? How would the reader find out?

Reply:
It’s a good observation but we do link to the RFC when we first introduce the term “PKCE”; all the detail can be read there. We’ve very deliberately not just rehashed the RFC to avoid creating a maintenance task